libs/discordURL.ts

```diff
@@ -0,0 +1,30 @@
+export const DISCORD_OAUTH_URL = 'https://discord.com/api/oauth2/token';
+
+export const discordRedirectBaseUrl = `${process.env.NEXT_PUBLIC_DISCORD_REDIRECT_URI_BASE}/api/discord/auth`
+
+export const discordScope = ["identify", "guilds"].join(" ");
+
+export const DISCORD_CLIENT_ID = process.env.NEXT_PUBLIC_DISCORD_CLIENT_ID
+
+export const discordAuthUrl = (from: string) => {
+  const url = "https://discord.com/api/oauth2/authorize" +
+  "?client_id=1093511877813870592&" +
+  `redirect_uri=${encodeURIComponent(`${discordRedirectBaseUrl}?from=${from}`)}&` +
+  `response_type=code&scope=${encodeURIComponent(discordScope)}`
+  return url
+}
+export const addBotUrl = (guildId: string) => {
+  return `https://discord.com/api/oauth2/authorize?client_id=${DISCORD_CLIENT_ID}&guild_id=${guildId}&permissions=326418033728&scope=bot`
```

Comment:
What does `3264...` here means? do you mind to add a comment?

Reply:
for the record, generated that number from this menu, will add explicit definitions from BIT FIELDS later on in case we need to to add more permissions

![image](https://github.com/nance-eth/nance-interface/assets/83923290/e8a317ae-2e85-411f-abe4-9de15ca7facc)